inbm/dispatcher-agent/dispatcher/aota/application_command.py

```diff
@@ -135,12 +151,23 @@ def update(self) -> None:
             # Move driver to CentOS filesystem
             move_file(driver_path, driver_centos_path)
 
+            old_driver_name = self.identify_package(driver_path.split('/')[-1])
+            if not old_driver_name:
+                raise AotaError(
+                    f'AOTA Command Failed: Unsupported driver {driver_path.split("/")[-1]}')
+            uninstall_driver_cmd = CHROOT_PREFIX + \
+                f'/usr/bin/rpm -e --nodeps {old_driver_name}'
+            out, err, code = PseudoShellRunner().run(uninstall_driver_cmd)
+
+            # If old packages wasn't install on system, it will return error too.
+            if code != 0 and "is not installed" not in str(err):
+                raise AotaError(err)
+
             chroot_driver_path = driver_centos_path.replace("/host", "")
             install_driver_cmd = CHROOT_PREFIX + \
```

Comment:
in the older RTC the install cmd that got replaced was "install_driver_cmd = DOCKER_CHROOT_PREFIX + f'**/usr/bin/rpm -ivh** {chroot_driver_path}'"

Are we not reverting back to this? I still see the -Uvh option being used.
Its probably not an issue as we are uninstalling the older version first and then calling the upgrade (-Uvh) cmd which may equate to an install (-ivh) cmd.